drupal8-example/Makefile

```diff
@@ -0,0 +1,20 @@
+up:
+ifeq ($(shell uname), Darwin)
+	docker-compose -f docker-compose.yml -f docker-compose-mac.yml up -d
+else
+	docker-compose up -d
+endif
+
+	# Tada!! 🎉 Run 'make logs' to follow output.
+
+down:
+ifeq ($(shell uname), Darwin)
+	docker-compose -f docker-compose.yml -f docker-compose-mac.yml stop
+	docker-compose -f docker-compose.yml -f docker-compose-mac.yml down -v --remove-orphans
+else
+	docker-compose stop
+	docker-compose down -v --remove-orphans
+endif
+
+logs:
+	@docker-compose logs -f
```

Comment:
Also why hide the command here?

Reply:
In a couple of months someone will add a service only for the mac setup. That services logs will not tailed here.

Have you considered moving the OS detecting out the individual targets and instead do it once and have it set just [COMPOSE_FILE environment file](https://docs.docker.com/compose/reference/envvars/#compose_file). It believe that could get us more readable, more maintainable,  and less error prone Makefiles.

Reply:
That's a good idea!

Reply:
Remember variables in Makedfiles are not exported by default so you'll need to do that explicitly:
```
export COMPOSE_FILE = docker-compose.yml:docker-compose-mac.yml
```
(also remember to consider impacts on the `docker-compose.override.yml`)

Reply:
Fixed

Reply:
Fixed? No, it's not. You took the idea from this approach and added it to the docker reset script(s). But, you did not use the approach when you moved the functionality into the other Makefile example.